src/execution/operator/csv_scanner/table_function/csv_file_scanner.cpp

```diff
@@ -16,31 +16,32 @@ CSVFileScan::CSVFileScan(ClientContext &context, shared_ptr<CSVBufferManager> bu
       error_handler(make_shared_ptr<CSVErrorHandler>(options_p.ignore_errors.GetValue())),
       on_disk_file(buffer_manager->file_handle->OnDiskFile()), options(options_p) {
 
+	auto global_columns =
+	    MultiFileReaderColumn::ColumnsFromNamesAndTypes(bind_data.return_names, bind_data.return_types);
```

Comment:
nit: same as for parquet, perhaps this should just be done once in the bind?

Reply:
Because of the `ReadCSVData::Deserialize` method I'd rather not mess with this
I could see the `columns` not getting populated somewhere, mainly there, because that code is generated